modules/virt-running-virtctl-ssh-command.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-running-virtctl-ssh-command_{context}"]
+= Running the virtctl SSH command
+
+Run the `virtcl ssh` command to access a running virtual machine (VM).
+
+.Prerequisites
+
+* You added a static SSH public key to the VM before it booted for the first time.
+
```

Comment:
The environment which runs the  `virtctl ssh` command has permissions on the OpenShift Cluster to access the VM.
(e.g. by executing `oc login` or setting KUBECONFIG like described in https://docs.openshift.com/container-platform/4.13/virt/virt-using-the-cli-tools.html#virt-installing-virtctl-client_virt-using-the-cli-tools ) This is worth to mention here, because the ssh connection via service or seconday network does not require this step.

Reply:
@dominikholler 
Is this "either/or"? Either the user logs in to the cluster OR sets KUBECONFIG? 

I'm not sure why a user would not have $KUBECONFIG set because it is part of the virtctl installation procedure.